compliance/collection/main.go

```diff
@@ -37,6 +41,31 @@ var (
 	once sync.Once
 )
 
```

Comment:
This file (`main.go`) size doubled with the introduction of all nodescan/nodeinventory stuff. It becomes harder to read and harder to reason about it, hence easy to make mistakes when maintaining. I know it's probably annoying, but could I ask you to move out internals of nodescan/nodeinventory that are not relevant to the high-level flow of `main.go` in a separate file, ideally package?

Reply:
Done in https://github.com/stackrox/stackrox/pull/4701/commits/3e6f0f9e9ba1b47a02ac1eb89b6ba359da80b5e7